routes.json

```diff
@@ -0,0 +1,3 @@
+{
+  "/api/v1/*": "/$1"
```

Comment:
What's the purpose of this?

Reply:
Sorry, I removed it,
It had no use there, 
I was using it before.